test/java/seedu/addressbook/logic/LogicTest.java

```diff
@@ -193,6 +193,7 @@ public void execute_list_showsAllPersons() throws Exception {
 
         // prepare address book state
         helper.addToAddressBook(addressBook, false, true);
+        saveFile.save(addressBook);
```

Comment:
Is this necessary? Do you have to save it here? Figure out where to test your changes. Notice that what method is called at end of each execute...() tests. Try to test isMutating() in that method.